CMakeLists.txt

```diff
@@ -159,9 +159,9 @@ find_package(Git QUIET)
 #   set(TACO_GIT_SHORTHASH "")
 # endif()
 
-if(NOT EXISTS "${TACO_PROJECT_DIR}/python_bindings/pybind11/CMakeLists.txt")
-  message(FATAL_ERROR "The submodules were not downloaded! GIT_SUBMODULE was turned off or failed. Please update submodules and try again.")
-endif()
+# if(NOT EXISTS "${TACO_PROJECT_DIR}/python_bindings/pybind11/CMakeLists.txt")
```

Comment:
Do you mind pulling this change out into a separate commit?

Reply:
Oops. Removed from this PR.